typhos/utils.py

```diff
@@ -73,12 +82,47 @@
 
 
 def _get_display_paths():
-    """Get all display paths based on PYDM_DISPLAYS_PATH + typhos built-ins."""
+    """
+    Get all display paths.
+
+    This includes, in order:
+
+    - The $PYDM_DISPLAYS_PATH environment variable
+    - The typhos.ui entry point
+    - typhos built-ins
+    """
     paths = os.environ.get('PYDM_DISPLAYS_PATH', '')
     for path in paths.split(os.pathsep):
         path = pathlib.Path(path).expanduser().resolve()
         if path.exists() and path.is_dir():
             yield path
+
+    _entries = entrypoints.get_group_all(TYPHOS_ENTRY_POINT_KEY)
+    entry_objs = []
+
+    for entry in _entries:
+        try:
+            obj = entry.load()
+        except Exception:
+            msg = (f'Failed to load {TYPHOS_ENTRY_POINT_KEY} '
+                   f'entry: {entry.name}.')
+            logger.error(msg)
+            logger.debug(msg, exc_info=True)
+            continue
+        if isinstance(obj, list):
+            entry_objs.extend(obj)
+        else:
+            entry_objs.append(obj)
+
+    for obj in entry_objs:
+        try:
+            yield pathlib.Path(obj)
+        except Exception:
+            msg = (f'{TYPHOS_ENTRY_POINT_KEY} entry point '
+                   f'{entry.name}: {obj} is not a valid path!')
+            logger.error(msg)
+            logger.debug(msg, exc_info=True)
```

Comment:
Random thought, but remember the `stacklevel` thing I was talking about the other day?

I think this is a pretty good example of where it'd be useful. We can now have:
`utils.log_error_and_debug_exception_message(logger, msg)` 
with `logger.error(msg, stacklevel=2)` and `logger.debug(msg, exc_info=True, stacklevel=2)` would indicate that the message came from this function.

That said, I think we're still 3.6+ on typhos, so ... a note for the future?